articles/how-to-use-array-operators-in-js/index.md

```diff
@@ -0,0 +1,507 @@
+
+```javascript
+let fruits = ['mangoes','lemons','bananas'];
+let new_fruits = fruits.slice(1,3);
+console.log(new_fruits);
+```
+Output
+
+```bash
+['lemons','bananas']
+```
+
+- **some()**: It checks if any of the data in the array meet a particular condition. For our fruits array we can check if there are some of the fruits which have eight letters. The `some()` method receives a function specifying the condition. If there are some, it shall return `true` otherwise `false`.
+
+```javascript
+let fruits = ['mangoes','lemons','bananas'];
+let exists = fruits.some(fruit => fruit.length === 8);
+console.log(exists);
+```
+Output
+
+```bash
+false
+```
+- **sort()**: This sorts the data inside an array. It receives two parameters which represent the data to be sorted. The function to be executed for sorting depends with the problem at hand. For example, using our fruits array we may decide to order them according to the number of letters  in ascending order using `sort()`.
+
+```javascript
+let fruits = ['mangoes','lemons','bananas'];
+let sorted_fruits = fruits.sort((fruit_a,fruit_b) => fruit_a.length - fruit_b.length);
+console.log(sorted_fruits);
+```
+Output
+
+```bash
+['lemons','mangoes','bananas']
+```
+
+- **splice()**: It removes elements from an array, replacing their positions with other elements of the array. When calling the `splice()`, you specify the start index and the `deleteCount`. If you omit the `deleteCount`, all of the elements in the array will be deleted. In our example, we delete two elements i.e the `deleteCount`, starting from index 0 i.e the start index.
+
+```javascript
+let fruits = ['mangoes','lemons','bananas'];
+let spliced_fruits = fruits.splice(0,2);
+console.log(spliced_fruits);
+console.log(fruits);
+```
+Output
+
+```bash
+['mangoes','lemons']
+['bananas']    
+```
+
+- **toString()**: It converts the target array to a string and returns the result. In our example, we can decide to convert our fruits array into a single string using `toString()`.
+
+```javascript
+let fruits = ['mangoes','lemons','bananas'];
+let _string = fruits.toString();
+console.log(_string)
+```
+Output
+
+```bash
+mangoes,lemons,bananas
+```
+
+- **unshift**: it is the opposite of `push()`. `unshift()` adds a new element to the beginning of the array and returns the new length. In our example, we can decide to push another fruit to the beginning of the array.
+
+```javascript
+let fruits = ['mangoes','lemons','bananas'];
+let new_length = fruits.unshift('apples');
+console.log(new_length);
+console.log(fruits);
+```
+Output
+
+```bash
+4
+['apples','mangoes','lemons','bananas']
+```
+
+### Conclusion
+In this article we have covered an introduction to JavaScript arrays, properties of JavaScript arrays and operations executed on JavaScript arrays.
```

Comment:
This conclusion seems to be a bit short. You could add more content regarding the problem the article aims o solve or what the topic addresses more. A conclusion with three sentences is ideal...

Reply:
solved